### PR TITLE
Jully 22 email

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2022-07-22.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+This is Tim Sehn (tim@dolthub.com), Founder and CEO of DoltHub. We decided this week to ditch the automated or semi-automated emails you all were getting as users. We're a tiny company. We want to build a personal connection with as many of you as possible. Automated emails weren't helping with that. I'm going to write this weekly email to you all every Friday until I can't, which I hope is many years from now.
+
+# That belongs in a museum...
+
+We're making great progess on the [museum collections bounty](https://www.dolthub.com/repositories/dolthub/museum-collections). We're up to 10.6M objetcs in `main`.
```
Comment (fulghum) [inline emails/2022-07-22.md:5]:
```suggestion
We're making great progess on the [museum collections bounty](https://www.dolthub.com/repositories/dolthub/museum-collections). We're up to 10.6M objects in `main`.
```

Comment (max-hoffman) [reply emails/2022-07-22.md]: "objects" is a little bland, "pieces of art, sculptures, and historical artifacts" or something?

File: emails/2022-07-22.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+```bash
+museum-collections $ dolt sql -q "select count(*) from objects"
++----------+
+| count(*) |
++----------+
+| 10659271 |
++----------+
+```
+
+We have approximately [20 outstanding Pull Requests](https://www.dolthub.com/repositories/dolthub/museum-collections/pulls) to review and merge. The top bounty hunter is [in line to make ~$3,400](https://www.dolthub.com/repositories/dolthub/museum-collections/bounties/880c0d21-c334-4417-bc18-8be7bddfb062/scoreboard). We continue to be inspired by the bounty process' ability to generate awesome open data to drive [interesting, socially relevant analysis](https://www.dolthub.com/blog/2022-07-01-hospitals-compliance/).
```
Comment (alecstein) [inline emails/2022-07-22.md:16]:
```suggestion
We have approximately [20 outstanding Pull Requests](https://www.dolthub.com/repositories/dolthub/museum-collections/pulls) to review and merge. The top bounty hunter is [in line to make ~$3,400](https://www.dolthub.com/repositories/dolthub/museum-collections/bounties/880c0d21-c334-4417-bc18-8be7bddfb062/scoreboard). We continue to be inspired by the bounty process's ability to generate awesome open data to drive [interesting, socially relevant analysis](https://www.dolthub.com/blog/2022-07-01-hospitals-compliance/).
```

File: emails/2022-07-22.md (hunk 1 above)
Comment (timsehn) [reply emails/2022-07-22.md]: Agreed

File: emails/2022-07-22.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# Released this week
+
+We released [0.40.18](https://github.com/dolthub/dolt/releases/tag/v0.40.18) and [0.40.19](https://github.com/dolthub/dolt/releases/tag/v0.40.19) this week. These two releases contain our usual cadence of bug fixes, small features, and usability improvements.
+
+The major user facing change is a change to the way we handle [privileges configuration](https://docs.dolthub.com/sql-reference/server/access-management). Before 0.40.18 you had to specify a privilege file location when you started a server to enable users and grants. This was a crappy user experience and caused confusion. Now, we create a `.doltcfg` directory and `privileges.db` file automatically when you create users and grants. Those familiar with MySQL know that privileges are stored in a `mysql` database not the user defined database. We preserve this behavior for compatibility. This means that privileges are not versioned.
```
Comment (alecstein) [inline emails/2022-07-22.md:22]:
```suggestion
The major user-facing change is to how we handle [privileges configuration](https://docs.dolthub.com/sql-reference/server/access-management). 

Before, to enable users and grants, you had to specify a privileges file. But in the Dolt 0.40.18, a `privileges.db` file is created for you automatically when you need it. To keep compatibility with MySQL, we use an un-versioned `mysql` database for privileges.
```

File: emails/2022-07-22.md (hunk 2 above)
Comment (timsehn) [reply emails/2022-07-22.md:16]: https://forum.wordreference.com/threads/process-or-processs.1704502/#:~:text=Thanks%20in%20advance!,-bj&text=The%20traditional%20way%20to%20form,method%20I%20prefer%20to%20use.

**Pull request merged**